3.evaluate_model/scripts/nbconverted/confusion_matrices.py

```diff
@@ -60,16 +65,42 @@
         
         # load dataset (train, test, etc)
         data = get_dataset(features_dataframe, data_split_indexes, label)
-        # path to save confusion matrix tidy data to
-        cm_save_path = pathlib.Path(f"{cm_dir}/{model_name}__{label}__cm.tsv")
-        
-        cm = model_cm(model, data)
+        # find confusion matrix for chosen model evaluated on chosen dataset      
+        cm_data = model_confusion_matrix(model, data)
         
+        # add confusion matrix data to compiled dataframe in tidy format
         # use stack to restructure dataframe into tidy long format
-        cm_tidy_data = cm.stack()
+        cm_data = cm_data.stack()
         # reset index must be used to make indexes at level 0 and 1 into individual columns
         # these columns correspond to true label and predicted label, and are set as indexes after using stack()
-        cm_tidy_data = pd.DataFrame(cm_tidy_data).reset_index(level=[0,1])
-        cm_tidy_data.columns = ["True_Label", "Predicted_Label", "Count"]
-        cm_tidy_data.to_csv(cm_save_path, sep="\t")
+        cm_data = pd.DataFrame(cm_data).reset_index(level=[0,1])
+        cm_data.columns = ["True_Label", "Predicted_Label", "Count"]
+        # add data split column to indicate which dataset scores are from (train, test, etc)
+        cm_data["data_split"] = label
+        # add shuffled column to indicate if the model has been trained with shuffled data (random baseline) or not
+        cm_data["shuffled"] = True if "shuffled" in model_name else False
```

Comment:
Consider using solely the `in` without the `if` and `else`.

Reply:
Done in https://github.com/WayScience/phenotypic_profiling_model/pull/9/commits/ce807b04f97a7ca330cb5223f4180bf8b61e2dfa